zulipterminal/model.py

```diff
@@ -1164,10 +1164,13 @@ def _handle_update_message_flags_event(self, event: Event) -> None:
                       self.controller, -1)
 
     def formatted_local_time(self, timestamp: int,
-                             *, show_seconds: bool) -> str:
+                             *, show_seconds: bool = False,
+                             show_year: bool = False) -> str:
         local_time = datetime.datetime.fromtimestamp(timestamp)
         if show_seconds:
             return local_time.strftime('%a %b %d %H:%M:%S')
+        if show_year:
```

Comment:
I was referring this addition. Notice that you're indirectly modifying this condition in the later commit (where you're declaring intermediate variable). However, since code-wise this is an insignificant change, I guess we shouldn't bother about discussing it more :)